src/coffee/directives/api/utils/model-key.coffee

```diff
@@ -35,8 +35,12 @@ angular.module('uiGmapgoogle-maps.directives.api.utils')
           return isEqual unless isEqual
         #compare the rest of the properties that are being watched by scope
         without = _.without(@interface.scopeKeys, 'coords')
+        console.log("Scope.deepcomparison = " + scope.deepComparison)
```

Comment:
Please use `$log.debug` instead of `console.log` that way the internal log is used and we don't spam a user's app unless they are explicitly debugging.